frontend/src/app/modules/external-booking/external-booking-page/external-booking-page.component.sass

```diff
@@ -0,0 +1,111 @@
+
+@import '../../../../styles/color-variables'
+@import '../../../../styles/screen-variables'
+
+*
+    box-sizing: border-box
+
+.container
+    display: flex
+    flex-direction: column
+    justify-items: center
+    background-color: $athens-gray
+    height: 100vh
+    width: 100vw
+    padding: 0 10% 5% 10%
+    @media screen and (max-width: $tablet-min-width)
+        padding: 0 5% 2% 5%
+
+.header
+    display: flex
+    flex-direction: row
+    justify-content: center
+    align-items: center
+    padding: 20px 0
+    img
+        width: 90px
+
+.main-content
+    display: flex
+    flex-direction: row
+    justify-content: center
+    padding: 0 20px
+    background-color: $white
+    height: 100vh
+    @media screen and (max-width: $desktop-min-width)
+        flex-direction: column
+    .sidebar
+        width: 25%
+        @media screen and (max-width: $desktop-min-width)
+            width: 100%
+            border-right: none
+            border-bottom: 2px solid $athens-gray
+    .choice
+        width: 75%
+        @media screen and (max-width: $desktop-min-width)
+            width: 100%
+
+.sidebar
+    position: relative
+    display: flex
+    flex-direction: column
+    align-items: flex-start
+    border-right: 2px solid $athens-gray
+    padding: 30px
+    img
+        height: 78px
+        width: 78px
+        border-radius: 100%
+        margin-bottom: 20px
+
+.choice
+    position: relative
+    display: flex
+    flex-direction: column
+    align-items: flex-start
+    padding: 30px 23px
+
+h1, h3, h4
+    font-family: 'Inter'
+    font-style: normal
+    line-height: 20px
+    color: $pale-sky
+
+h3, h4
+    font-weight: 400
+    font-size: 14px
+
+h1
+    font-weight: 700
+    font-size: 20px
+
+::ng-deep .mat-card
+    width: 45%
+    height: 101px
+    margin: 5px 0
+    padding-top: 20px !important
+    padding-left: 20px !important
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.16) !important
+    border-radius: 4px !important
+    @media screen and (max-width: $desktop-min-width)
+        width: 100%
+
+::ng-deep .mat-card:hover
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25) !important
+    cursor: pointer
+
+::ng-deep .mat-card-title, .mat-card-subtitle
+    font-family: 'Inter' !important
+    font-style: normal
+    line-height: 20px
+    color: $pale-sky
+
+::ng-deep .mat-card-title
+    font-weight: 700 !important
+    font-size: 20px !important
+    margin-top: 5px !important
+
+::ng-deep .mat-card-subtitle
+    font-weight: 400 !important
+    font-size: 14px !important
+    margin-top: 10px !important
```

Comment:
Are you sure this will not override existing styles?

Reply:
Yes